src/twisted/python/_setup.py

```diff
@@ -350,10 +355,14 @@ def _checkCPython(sys=sys, platform=platform):
 notPortedModules = [
     "twisted.conch.client.connect",
     "twisted.conch.client.direct",
+    "twisted.conch.tap",
     "twisted.conch.test.test_cftp",
     "twisted.conch.test.test_ckeygen",
     "twisted.conch.test.test_conch",
+    "twisted.conch.test.test_helper",
```

Comment:
this actually got ported [recently](https://github.com/twisted/twisted/pull/554)